File: JSON2/idm.php
<?php
	$voiture = array();
	$voiture["Annee"] = "2010";
	$voiture["Marque"]  = "Volvo";
	$voiture["Puissance"] = "190cv";
	$voiture["Modele"]  = "xc90";
	echo json_encode( $voiture );
?>

File: JSON2/idm.java
import org.json.simple.JSONObject;

public class idm {

	public static void main(String[] args) {
		   JSONObject voiture = new JSONObject();
		   voiture.put("Annee", "2010");
		   voiture.put("Marque", "Volvo");
		   voiture.put("Puissance", "190cv");
		   voiture.put("Modele", "xc90");
		   System.out.print(voiture);
	}
}
